### PR TITLE
Add 1inch v4 (Own, Uniswap V2 router, Uniswap V3 router, Clipper)

### PR DESCRIPTION
**Pull request by k06a**
I've checked that:

* [x] the query produces the intended results
* [x] the folder name matches the schema name
* [x] the schema name exists in Dune
* [x] views are prefixed with `view_`, functions with `fn_`.
* [x] the filename matches the defined view, table or function and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

### CODE REVIEW

File: ethereum/dex/trades/insert_1inch.sql [hunk 1]
```diff
@@ -123,13 +123,88 @@ WITH rows AS (
             call_trace_address AS trace_address,
             NULL::integer AS evt_index
         FROM (
-            select "output_returnAmount", "amount", "srcToken", "_3" as pools, "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" from oneinch_v3."AggregationRouterV3_call_unoswap" union all
-            select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" from oneinch_v3."AggregationRouterV3_call_unoswapWithPermit"
+            select "output_returnAmount", "amount", "srcToken", "_3" as pools, "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" from oneinch_v3."AggregationRouterV3_call_unoswap" when call_success union all
```
Comment (masquot) [inline ethereum/dex/trades/insert_1inch.sql:126]: On this line and the following ones, probably `when` needs to be `WHERE`?

File: ethereum/dex/trades/insert_1inch.sql [hunk 2]
```diff
@@ -123,13 +123,88 @@ WITH rows AS (
...
+            select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" from oneinch_v3."AggregationRouterV3_call_unoswapWithPermit" when call_success
+            UNION ALL
+            select "output_returnAmount", "amount", "srcToken", "_3" as pools, "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" from oneinch_v4."AggregationRouterV4_call_unoswap"  when call_success union all
```
Comment (masquot) [inline ethereum/dex/trades/insert_1inch.sql:129]: This line gives the following error when trying to run it:
```
Column "_3" does not exist
```

**Review by masquot — CHANGES_REQUESTED** [review]

**Comment by k06a** [comment]
@masquot applied fixes, sorry about this

**Comment by masquot** [comment]
No worries, thank you @k06a for all your great contributions! Could it be that I'm not seeing the new commit?

How far in time should the backfill go? Noticed you published the blog post about *The 1inch Router v4 Roll-out* on 10 November.

**Comment by k06a** [comment]
@masquot sorry, pushed to the wrong branch

**Comment by k06a** [comment]
@masquot yep, but trades started a bit earlier, because we tested it :) So you can use smart contract deployment date

**Review by masquot — APPROVED** [review]

**Pull request merged**